src/style.css

```diff
@@ -1 +1,25 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url(images/star.svg) center no-repeat;
```

Comment:
Consider adding a fallback background color for the `.stars__star` in case the image fails to load, to maintain an acceptable user experience.